test/test_nn.py

```diff
@@ -8010,6 +8010,20 @@ def test_RNN_dropout(self):
                     self.assertEqual(hy.data[0][0][0], 10)
                     self.assertEqual(hy.data[1][0][0], output_val)
 
+    @unittest.skipIf(not (TEST_CUDNN and (TEST_CUDNN_VERSION if TEST_CUDNN_VERSION else 0) >= 5103), "needs cudnn >= 5.1")
```

Comment:
We don't need the skipIf: RNNs still work without CUDNN

Reply:
sounds good, as our offline discussion, i made the suggested change